docs/develop/nova/block_explorer.md

```diff
@@ -1,18 +1,17 @@
----
-title: Subspace block explorer
-sidebar_position: 10
-description: Subspace hosted block explorer
-keywords:
-  - block explorer
-  - subspace network
----
+# Subspace block explorer
```

Comment:
Let's keep the section at the top unchanged as it helps with navigation and SEO. Thanks!

Reply:
okkay cool